src/worker.js

```diff
@@ -54,16 +39,19 @@ create().onRequest('job', ({ contents, type, config, filePath, projectPath, rule
   const configPath = Helpers.getConfigPath(fileDir)
   const relativeFilePath = Helpers.getRelativePath(fileDir, filePath, config)
 
-  const argv = Helpers.getArgv(type, config, rules, relativeFilePath, fileDir, configPath)
+  const cliEngineOptions =
+    Helpers.getCLIEngineOptions(type, config, rules, relativeFilePath, fileDir, configPath)
```

Comment:
I'm not sure how @Arcanemagus feels about this one, but personally I would prefer avoiding the long-line error by formatting it as:

```js
const cliEngineOptions = Helpers.getCLIEngineOptions(
  type, config, rules, relativeFilePath, fileDir, configPath
)
```